conversation_event.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type EventParams struct {
```

Comment:
Just to keep the godocs complete 😉 

```suggestion
// EventParams are the parameters to Client.AddConversationEvents.
type EventParams struct {
```